devicetypes/smartthings/ecobee-thermostat.src/ecobee-thermostat.groovy

```diff
@@ -156,47 +156,49 @@ void poll() {
 def generateEvent(Map results) {
 	log.debug "parsing data $results"
 	if(results) {
-		results.each { name, value ->
 
 			def linkText = getLinkText(device)
-			def isChange = false
-			def isDisplayed = true
+		def supportedThermostatModes = []
+		def thermostatMode = null
+
+		results.each { name, value ->
 			def event = [name: name, linkText: linkText, descriptionText: getThermostatDescriptionText(name, value, linkText),
 						 handlerName: name]
 
 			if (name=="temperature" || name=="heatingSetpoint" || name=="coolingSetpoint" ) {
 				def sendValue =  location.temperatureScale == "C"? roundC(convertFtoC(value.toDouble())) : value.toInteger()
-				isChange = isTemperatureStateChange(device, name, value.toString())
-				isDisplayed = isChange
-				event << [value: sendValue, unit: temperatureScale, isStateChange: isChange, displayed: isDisplayed]
+				event << [value: sendValue, unit: temperatureScale]
```

Comment:
Just curious why we don't care anymore about whether or not the value of the event is changed, and why we don't send isDisplayed anymore.